### DISCUSSION TITLE
how to pull images with insecure registry

### DISCUSSION THREAD

**Discussion by daxiguafan** [bazelbuild/rules_docker#2050] [Q&A]
In my test env. I use a insecure registry to speed up. But i find puller failed with "http: server gave HTTP response to HTTPS client".
How to set insecure_registry option?

**Comment by cfife-btig** [comment]
I think this is something that doesn't exist at the moment. I would really like this feature as well

I think this might be an PR of interest https://github.com/bazelbuild/rules_docker/pull/1403
